hack/rp-dev/preRPDeploy.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash -e
+
+source ./hack/devtools/deploy-shared-env.sh
+deploy_vpn_for_dedicated_rp
+echo "Success step 4a 🚀 - VPN has been deployed"
+
+deploy_aks_dev
+echo "Success step 4b 🚀 - AKS has been deployed"
+echo "Success step 4 ✅ - VPN & AKS have been deployed"
+
+vpn_configuration
+screen -dmS connect_dev_vpn bash -c "sudo openvpn secrets/vpn-$LOCATION.ovpn; sleep 10; exec bash" # open new socket to run concurrently
+make aks.kubeconfig
+HOME=/usr ./hack/hive-generate-config.sh # HOME is default to /usr which isn't in PATH
+KUBECONFIG=$(pwd)/aks.kubeconfig ./hack/hive-dev-install.sh
+echo "Success step 5 ✅ - Hive has been installed"
+
+export DST_ACR_NAME=${AZURE_PREFIX}aro
+export SRC_AUTH_QUAY=$(echo $USER_PULL_SECRET | jq -r '.auths."quay.io".auth')
+export SRC_AUTH_REDHAT=$(echo $USER_PULL_SECRET | jq -r '.auths."registry.redhat.io".auth')
+export DST_AUTH=$(echo -n '00000000-0000-0000-0000-000000000000:'$(az acr login -n $DST_ACR_NAME --expose-token | jq -r .accessToken) | base64 -w0)
+docker login -u 00000000-0000-0000-0000-000000000000 -p "$(echo $DST_AUTH | base64 -d | cut -d':' -f2)" "$DST_ACR_NAME.azurecr.io"
```

Comment:
I haven't figured this one out yet, but my deploy fails here with the following:
```
Error: failed to open 2048 locks in /libpod_lock: permission denied
```

Reply:
Following up - this issue seemed to go away for me by moving to a rootful podman machine (podman machine init --rootful)